src/Integration.UnitTests/Binding/CSharpVBBindingConfigProviderTests.cs

```diff
@@ -196,13 +209,15 @@ public async Task GetConfig_HasActiveInactiveAndUnsupportedRules_ReturnsValidBin
             var activeRules = new SonarQubeRule[]
                 {
                     CreateRule("activeRuleKey", "repoKey1", true),
-                    CreateRule("activeUnsupported", "roslyn.sonaranalyzer.security.foo", true)
+                    ActiveTaintAnalysisRule,
+                    ActiveRuleWithUnsupportedSeverity
```

Comment:
... and the changes to this test make sure we check unsupported issue types are filtered out of both active and inactive rules.